tests/fuzzers/les/debug/main.go

```diff
@@ -32,9 +33,16 @@ func main() {
 		os.Exit(1)
 	}
 	crasher := os.Args[1]
-	data, err := ioutil.ReadFile(crasher)
+
+	canonicalPath, err := common.VerifyPath(crasher)
```

Comment:
could we have 1 function for it and remove copy-pasting?

Reply:
Hey @JekaMas could you please elaborate more? Didn't quite get it.
I am already using a function defined in the `common` package (`common.VerifyPath(...)`)

Reply:
this snippet:
```
canonicalPath, err := common.VerifyPath(crasher)
if err != nil {
		fmt.Println("path not verified: " + err.Error())
		return
	}

data, err := ioutil.ReadFile(canonicalPath)
```
is copied in 7 places.

Reply:
Fixed at commit [860f74db67](https://github.com/maticnetwork/bor/pull/578/commits/860f74db67c2da94f2d12faca2115bcb1642e3fa)

Thanks

Reply:
Ia the fix ok for you, @JekaMas ? 
Thanks